sdk/python/feast/feature_view.py

```diff
@@ -86,6 +87,9 @@ class FeatureView(BaseFeatureView):
         tags: A dictionary of key-value pairs to store arbitrary metadata.
         owner: The owner of the feature view, typically the email of the primary
             maintainer.
+        mode: The transformation mode for feature transformations. Only meaningful when
```

Comment:
@franciscojavierarceo As discussed, moved mode to FeatureView level aligning with unification to have mode in both FeatureView and Transformations.  https://github.com/feast-dev/feast/pull/5690/commits/a1fa595c104e919401f1f8628812d8cb8c9b2379 

```
FeatureView(
   transform=udf(stuff),
   engine="spark",
   mode="pandas",
   online=True, 
)
```